docs/architecture.md

```diff
@@ -15,14 +15,27 @@ Find below an overview of requirements, versions and supported configurations.
 - Uses Maven 3+ as build system
 
 ## Versions and supported configuration
-SDLB currently maintains the following two major versions, which are published as Maven artifacts on maven central:
+SDLB is published as Maven artifacts on Maven Central. 
+The SDLB versions are build with specific versions of Apache Spark and other libraries.
+This page gives an overview of the respective versions.
 
-|SDL Version|Java/Scala/Hadoop Version|File Engine|Spark Engine|Snowflake/Snowpark Engine|Comments|
-| --------- | ----------------------- | --------- | ---------- | ----------------------- | ------ |
-|1.x, branch master/develop-spark2|Java 8, Scala 2.11 & 2.12|Hadoop 2.7.x|Spark 2.4.x|not supported|Delta lake has limited functionality in Spark 2.x|
-|2.x, branch master/develop-spark3|Java 8+, Scala 2.12|Hadoop 3.3.x (2.7.x)|Spark 3.2.x (3.1.x)|Snowpark 1.2.x|Delta lake, spark-snowflake and spark-extensions need specific library versions matching the corresponding spark minor version|
+### SDLB Version 1.X
+SDLB version 1.X used Apache Spark 2.X. 
+This branch of SDLB is no longer maintained.
+To profit from the latest development, please upgrade to a more recent version of SDLB.
+
+### SDLB Version 2.X
+
+The following table gives an overview of dependency versions that are delivered with each major branch of SDLB.
+
+| SDL Version | Java/Scala/Hadoop Version | Hadoop Version | Spark Engine     | Snowflake/Snowpark Engine | Delta Lake  |
```

Comment:
Can we also add Iceberg and Log4j versions?
SDL 2.5+ -> log4j 2.x, before it was log4j 1.2.x

Reply:
Yes we can, good idea:
https://github.com/smart-data-lake/smart-data-lake/pull/718/commits/91fb771b580c133a0630f7a4cc52f05b932de36b